.vscode/launch.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
i must have missed this in the first pr, but this shouldn't be included. we can remove this as a hotfix.